napari/_qt/qt_dims.py

```diff
@@ -251,3 +262,9 @@ def collapse_change_listener(collapsed):
         slider.collapsedChanged.connect(collapse_change_listener)
 
         return slider
+
+    def _update_display(self):
```

Comment:
i presume your intention here is to trim the number of sliders based on the number of displayed axes. To do this you'll want to check if the `-3` slider is present in the list of currently displayed sliders and if so call `_remove_slider(index)` with the `index`. I can help with the details, especially if we want to make sure it can be added back in

Reply:
fixed